<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LotPackWindow</class>
 <widget class="QMainWindow" name="LotPackWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LotPack Viewer</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="margin">
     <number>2</number>
    </property>
    <item row="0" column="0">
     <widget class="LotPackView" name="view"/>
    </item>
    <item row="1" column="0">
     <layout class="QHBoxLayout" name="statusBarLayout">
      <item>
       <widget class="QLabel" name="coords">
        <property name="text">
         <string>Cell X,Y</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="mappingStatus">
        <property name="text">
         <string>Mapping</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel">
        <property name="text">
         <string>Startcell X:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="cellStartX">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>0</height>
         </size>
        </property>
        <property name="minimum">
         <number>0</number>
        </property>
        <property name="maximum">
         <number>1000</number>
        </property>
				<property name="value">
					<number>0</number>
				</property>
       </widget>
      </item>
      <item>
       <widget class="QLabel">
        <property name="text">
         <string>Y:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="cellStartY">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>0</height>
         </size>
        </property>
        <property name="minimum">
         <number>0</number>
        </property>
        <property name="maximum">
         <number>1000</number>
        </property>
				<property name="value">
					<number>0</number>
				</property>
       </widget>
      </item>
      <item>
       <widget class="QLabel">
        <property name="text">
         <string>Width:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="numCellX">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>0</height>
         </size>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>1000</number>
        </property>
				<property name="value">
					<number>3</number>
				</property>
       </widget>
      </item>
      <item>
       <widget class="QLabel">
        <property name="text">
         <string>Height:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="numCellY">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>0</height>
         </size>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>1000</number>
        </property>
				<property name="value">
					<number>20</number>
				</property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QComboBox" name="scaleCombo">
        <property name="editable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen_World"/>
    <addaction name="separator"/>
    <addaction name="actionRecent"/>
    <addaction name="separator"/>
    <addaction name="actionClose"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionCenterOnCell"/>
    <addaction name="separator"/>
    <addaction name="actionShowMiniMap"/>
    <addaction name="actionShowRoomDefs"/>
    <addaction name="separator"/>
    <addaction name="actionHighlightCurrentLevel"/>
    <addaction name="actionLevelUp"/>
    <addaction name="actionLevelDown"/>
    <addaction name="separator"/>
    <addaction name="actionZoomIn"/>
    <addaction name="actionZoomOut"/>
    <addaction name="actionZoomNormal"/>
    <addaction name="separator"/>
    <addaction name="actionSaveScreenshot"/>
    <addaction name="actionStartMapping"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuView"/>
  </widget>
  <action name="actionOpen_World">
   <property name="icon">
    <iconset resource="editor.qrc">
     <normaloff>:/images/16x16/document-open.png</normaloff>:/images/16x16/document-open.png</iconset>
   </property>
   <property name="text">
    <string>Open World...</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>Close</string>
   </property>
  </action>
	<action name="actionSaveScreenshot">
		<property name="text">
			<string>Save Screenshot</string>
		</property>
	</action>
	<action name="actionStartMapping">
		<property name="text">
			<string>Start Mapping</string>
		</property>
	</action>
  <action name="actionZoomIn">
   <property name="icon">
    <iconset resource="editor.qrc">
     <normaloff>:/images/16x16/zoom-in.png</normaloff>:/images/16x16/zoom-in.png</iconset>
   </property>
   <property name="text">
    <string>Zoom In</string>
   </property>
  </action>
  <action name="actionZoomOut">
   <property name="icon">
    <iconset resource="editor.qrc">
     <normaloff>:/images/16x16/zoom-out.png</normaloff>:/images/16x16/zoom-out.png</iconset>
   </property>
   <property name="text">
    <string>Zoom Out</string>
   </property>
  </action>
  <action name="actionZoomNormal">
   <property name="icon">
    <iconset resource="editor.qrc">
     <normaloff>:/images/16x16/zoom-original.png</normaloff>:/images/16x16/zoom-original.png</iconset>
   </property>
   <property name="text">
    <string>Normal Size</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+0</string>
   </property>
  </action>
  <action name="actionShowRoomDefs">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show RoomDefs</string>
   </property>
   <property name="shortcut">
    <string>R</string>
   </property>
  </action>
  <action name="actionCenterOnCell">
   <property name="text">
    <string>Center On Cell...</string>
   </property>
  </action>
  <action name="actionHighlightCurrentLevel">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Highlight Current Level</string>
   </property>
   <property name="shortcut">
    <string>H</string>
   </property>
  </action>
  <action name="actionLevelUp">
   <property name="text">
    <string>Select Level Above</string>
   </property>
   <property name="shortcut">
    <string>PgUp</string>
   </property>
  </action>
  <action name="actionLevelDown">
   <property name="text">
    <string>Select Level Below</string>
   </property>
   <property name="shortcut">
    <string>PgDown</string>
   </property>
  </action>
  <action name="actionRecent">
   <property name="text">
    <string>Recent</string>
   </property>
  </action>
  <action name="actionShowMiniMap">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show MiniMap</string>
   </property>
   <property name="shortcut">
    <string>M</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>LotPackView</class>
   <extends>QGraphicsView</extends>
   <header>lotpackwindow.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="editor.qrc"/>
 </resources>
 <connections/>
</ui>
